docs/en/05_Releases_and_changelogs/cwp_2.6.0.md

```diff
@@ -26,6 +26,22 @@ This CWP release includes an update to the [fulltextsearch module](https://githu
 
 If your website requires draft content to be indexed, you can [opt-out](https://github.com/silverstripe/silverstripe-fulltextsearch/blob/3/README.md#important-note-when-upgrading-to-fulltextsearch-37) of the new secure defaults.
 
+### Subsites is no longer included in the default CWP installation
+
+The [Subsites module](https://github.com/silverstripe/silverstripe-subsites) has been removed from
+[cwp/installer](https://github.com/silverstripe/cwp-installer) in 2.6.0, in order to reduce the
+default technical footprint of the recipe. Many sites operate without need of Subsites, and we feel
+that including this module should be left as an explicit decision for developers to make.
+
+New projects that need Subsites will need to manually install it via Composer:
```

Comment:
might be worth mentioning explicitly that it does not affect the existing/upgrading projects

Reply:
I did have this language originally, but it _could_ affect existing projects though - we're removing it from the upgrading instructions, so developers need to make sure they continue to require it themselves (which I mention below.)

Reply:
Oh, fair enough. I didn't notice that it says about existing projects, read as `if you're going to install it` :D
Good to merge then.